INSTALL.md

```diff
@@ -0,0 +1,291 @@
+
+The location of the apache configuration file is dependent on the operating system.  In linux, you can type `httpd -V` or `apache2ctle  -V` on the commandline;  the location to the configuration file will be the `HTTPD_ROOT` variable plus the `SERVER_CONFIG_FILE` variable. In Windows, the` XAMPP 1.7.0` package places the configuration file at `xampp\apache\conf\httpd.conf`. 
+
+To configure Zend and to secure the `/documents`, `/edi` and `/era` directories you can paste following to the end of the apache configuration file (ensure you put full path to directories):
+```xml
+<Directory "libreehrwebroot">
+AllowOverride FileInfo
+</Directory>
+<Directory "libreehrwebroot/sites">
+AllowOverride None
+</Directory>
+<Directory "libreehrwebroot/sites/*/documents">
+order deny,allow
+Deny from all
+</Directory>
+<Directory "libreehrwebroot/sites/*/edi">
+order deny,allow
+Deny from all
+</Directory>
+<Directory "libreehrwebroot/sites/*/era">
+order deny,allow
+Deny from all
+</Directory>
+```
+
+The final screen includes some additional instructions and important information. We suggest you print these instructions for future reference.
+
+Once the system has been configured properly, you may login.  Connect to the webserver where the files are stored with your web browser.  Login to the system using the username that you picked (default is `admin`), and the password.  From there, select the `Administration` option, and customize the system to your needs.  Add users and groups as is needed. For information on using LibreHealthEHR, consult the User Documentation located in the `Documentation` folder, the documentation at [LibreHealth](http://librehealth.io/).
+
+Reading `libreehr/sites/default/config.php` is a good idea.
+
+To create custom encounter forms online documentation at [LibreHealth](http://librehealth.io/). Many forms exist in `interface/forms` and may be used as examples.
+
+Other configuration settings are stored under `includes/config.php`. Everything should work out of the installation without touching those, but if  you want fax integration you will need to adjust some parameters in that file.
+
+General-purpose fax support requires customization within LibreHealthEHR at Administration->Globals and custom/faxcover.txt; it also requires the following utilities:
+
+* faxstat and sendfax from the HylaFAX client package
+* mogrify from the ImageMagick package
+* tiff2pdf, tiffcp and tiffsplit from the libtiff-tools package
+* enscript
+
+
+##   Setting Up Access Control
+
+Since LibreHealthEHR version `1.01`, phpGACL access controls are installed and configured automatically during LibreHealthEHR setup.  It can be administered within LibreHealthEHR in the admin->acl menu.  This is very powerful access control software. 
```

Comment:
Should we take out the"Since LibreHealthEHR version `1.01`" and just say "phpGACL access controls are installed and configured automatically during LibreHealthEHR setup."